docs/application/dotnet/guides/nui/visuals.md

```diff
@@ -400,239 +507,267 @@ The following features are not supported:
+   _visualView.AddVisual("Animated-Image", _animatedVisual);
+   ```
 
-    **Table: Shading mode values**
+The final output after implementing the preceding code is as follows:
 
-    | Enumeration value                      | Description                              |
-    |--------------------------------------|----------------------------------------|
-    | `TexturelessWithDiffuseLighting`       | Simplest, one color that is lit by ambient and diffuse lighting. |
-    | `TexturedWithSpecularLighting`         | Uses only the visual image textures provided with specular lighting in addition to ambient and diffuse lighting. |
-    | `TexturedWithDetailedSpecularLighting` | Uses all textures provided including a gloss, normal, and texture map along with specular, ambient, and diffuse lighting. |
+![Animated_Image_Visual](media/animated-image-visual.gif)
 
-**Usage:**
+### MeshVisual
 
-The following example shows how to use a `MeshVisual` visual map. The visual is created in the `AddVisual()` method.
+The `MeshVisual` renders a mesh using an `.obj` file, optionally with materials provided in an `.mtl` file and textures stored in the provided directory.
 
-```
-MeshVisual meshVisualMap1 = new MeshVisual();
+> [!NOTE]
+> Due to a typographical mistake in the source code, it is advised to use `MaterialtURL` instead of `MaterialURL` as the property name.
 
-meshVisualMap1.ObjectURL = resources + "/models/Dino.obj";
-meshVisualMap1.MaterialtURL = resources + "/models/Dino.mtl";
-meshVisualMap1.TexturesPath = resources + "/images/";
-meshVisualMap1.ShadingMode = MeshVisualShadingModeValue.TexturedWithSpecularLighting;
+The following table lists the supported properties:
 
-meshVisualMap1.Size = new Size2D(400, 400);
-meshVisualMap1.Position = new Position2D(-50, 600);
-meshVisualMap1.PositionPolicy = VisualTransformPolicyType.Absolute;
-meshVisualMap1.SizePolicy = VisualTransformPolicyType.Absolute;
-meshVisualMap1.Origin = Visual.AlignType.TopBegin;
-meshVisualMap1.AnchorPoint = Visual.AlignType.TopBegin;
+**Table: MeshVisual properties**
 
-_visualView.AddVisual("meshVisual1", meshVisualMap1);
-```
+| Property         | Type                | Required              | Description                                                                                                                       |
+|------------------|---------------------|-----------------------|-----------------------------------------------------------------------------------------------------------------------------------|
+| `ObjectURL`      | `string`            | Yes                   | The location of the `.obj` file.                                                                                                  |
+| `MaterialtURL`   | `string`            | No                    | The location of the `.mtl` file. Leave blank for a textureless object.                                                            |
```

Comment:
If fixing at line 565, kindly fix here as well.

The location of the MTL file.